instrumentation/methods/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/methods/MethodInstrumentation.java

```diff
@@ -91,42 +92,46 @@ public void transform(TypeTransformer transformer) {
   @SuppressWarnings("unused")
   public static class MethodAdvice {
 
+    public static class AdviceLocals {
```

Comment:
Should we be consistent with the others?

```suggestion
    public static class AdviceScope {
```